src/execution_plan/ops/shared/print_functions.c

```diff
@@ -9,10 +9,12 @@
 
 int TraversalToString(const OpBase *op, char *buf, uint buf_len, AlgebraicExpression *ae) {
 	int offset = 0;
+	offset += snprintf(buf, buf_len, "%s", op->name);
```

Comment:
Were the changes in this file intentional? This generates explain/profile output like:
```
1) "Results"
2) "    Project"
3) "        Conditional Traverse(a)-[r1]->(b)"
4) "            All Node Scan | (a)"
```

Reply:
Good call, thanks!